OWNERS_ALIASES

```diff
@@ -192,3 +192,10 @@ aliases:
   #
   sig-architecture-approvers: []
   sig-architecture-reviewers: []
+
+  #
+  # SIG Control plane
+  # Responsible for the development and enhancement of
+  # KubeVirt control plane.
+  sig-control-plane-approvers: []
+  sig-control-plane-reviewers: []
```

Comment:
@dasionov @Barakmor1 - Any objections to adding you both as reviewers here?

Reply:
No objections :)